packages/babel/src/transformation/file/options/config.json

```diff
@@ -240,6 +236,12 @@
     "description": "set `sources[0]` on returned source map"
   },
 
+  "inputSourceMap": {
+    "type": "boolean",
+    "description": "use existing source map for source map generation - only works with inline source maps",
```

Comment:
I understand the disconnect now.

I now understand #2336 to mean that source map merging is broken and you are correct that this change does not address that.  I misunderstand by what it was meant to be fixed.

Despite that, if the source map merging is not exposed on the CLI, what is the desired way to access/use this feature?  I think I missed how I should be appropriately be setting this option.